cmd/celvet/main.go

```diff
@@ -81,23 +92,58 @@ func main() {
 	}
 
 	limitErrors := celvet.CheckMaxLimits(structural)
+	costErrors, compileErrors, otherErrors := celvet.CheckExprCost(structural)
+	if useJSON {
+		emitJSON(limitErrors, costErrors, compileErrors, otherErrors)
+	} else {
+		emitText(limitErrors, costErrors, compileErrors, otherErrors, *humanReadable)
+	}
+
+	if len(limitErrors)+len(costErrors)+len(compileErrors) > 0 {
+		os.Exit(1)
+	}
+}
+
+func emitJSON(limitErrors []error, costErrors []*celvet.CostError, compileErrors []*celvet.CompilationError, otherErrors []error) {
+	type JSONOutput struct {
+		LimitErrors   []error                    `json:"limitErrors"`
+		CostErrors    []*celvet.CostError        `json:"costErrors"`
+		CompileErrors []*celvet.CompilationError `json:"compileErrors"`
+		OtherErrors   []string                   `json:"otherErrors"`
```

Comment:
What is OtherErrors for?  Is there a more descriptive name that could be used?  Is this for linter failures?

Reply:
`OtherErrors` is for errors returned by `cel.Compile` that aren't related to compilation errors, so it's for internal/non-linting-related failures. Maybe `InternalErrors` would be a better name?

Reply:
(marking this as fixed this since `OtherErrors` has been renamed to `InternalErrors`)